docs/faq.rst

```diff
@@ -514,7 +514,8 @@ The Borg config directory has content that you should take care of:
   recovered.
 
 ``keys`` subdirectory
-  In this directory all your repository keyfiles are stored. You MUST make sure to have an
+  In this directory all your repository keyfiles are stored. This does **not** apply if you
+  use the repokey mode because your key is stored alongside the the repository. You MUST make sure to have an
```

Comment:
almost. :-)

I'ld rather phrase it like "This does not apply to repokey keys because for them the key is stored in the repository."

Note: There still can be other keys in that keys subdirectory.